source/support/data-protection/index.html.md.erb

```diff
@@ -0,0 +1,162 @@
+---
+title: Data Protection for GOV.UK Account users
+weight: 32
+last_reviewed_on: 2021-02-02
+review_in: 6 months
+---
+
+# Respond to data protection requests
+
+When we use and hold user data, users gain rights under the UK General Data Protection Regulation (UK GDPR).
+Users make requests to exercise those rights and we have responsibilities to respond.
+
+Below are some of the scenarios we will have to to understand and respond to along with information on who
+has responsibilities.
+
+Responding to requests may involve the GOV.UK Accounts team, GOV.UK User support and the GDS Privacy office.
+
+This guide is not an complete list of everything we must do under the UK GDPR. Instead it focuses on user rights
+requests and how to respond.
+
+# Who handles requests?
+
+Responding to rights requests is a collaboration between the Privacy Office, User support team and GOV.UK Accounts team.
+All requests should initially arrive through zendesk where they can be flagged by User Support.
+
+If a request arrives directly with the accounts team they should create a new zendesk ticket. Where possible the ticket should contain:
+- Contact details for the user who has made the request
+- What their request is
+- When their request arrived with the GOV.UK account team
+- @TODO (any tags?)
+
+It can then be assigned to @TODO (???)
+
+Privacy office will work with user support to ensure the request is valid. If the request requires further actions from
+the accounts team user support will @TODO (How will they make contact?)
+
+# Time limits
+
+Collectively GOV.UK will have 30 calendar days to respond to any rights request. The clock does not stop for non-working days such as bank holidays or weekend.
+
+Where the Accounts team receives a request we should assign it to user support as soon as we see it.
+
+If user support asks accounts to provide information or take an action as part of a rights request, we should respond to user support within 7 days.
+
+# Self service
```

Comment:
Slightly unsure how this fits into the overall process? Are we saying that we should only take action when the user cannot self-serve?

Reply:
@jonathanglassman I gave this section another go.
What do you think?

Basically trying to day:
- good news, most users will be able to handle their own requests
- if a user contacts us, we've set up some template replies to show them how if they're unclear
- however if they can't or don't want to, we'd have to help them under most cirucumstances (probably not best to go into those... ask privacy office is the answer to most things there?)

Reply:
Looks good to me :)